.lando.yml

```diff
@@ -12,6 +12,9 @@ services:
     core: orangelight-core-dev
     config:
       dir: "solr/conf"
+  orangelight_database:
+    type: postgres:10
+    portforward: true
```

Comment:
can we use postgres 12?

Reply:
Is that what's in prod?

Reply:
yes

Reply:
Wait. no in production is 10. I was looking the qa it has 12.

Reply:
Yeah, `PostgreSQL 10.14 (Ubuntu 10.14-1.pgdg16.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609, 64-bit`